src/Buckle/Repl/BelteRepl.cs

```diff
@@ -382,12 +383,69 @@ internal sealed class BelteReplState {
         compilation.EmitTree(symbol, Console.Out);
     }
 
-    [MetaCommand("exit", "Exits the repl")]
+    [MetaCommand("exit", "Exit the repl")]
     private void EvaluateExit() {
         Environment.Exit(0);
     }
 
-    [MetaCommand("settings", "Opens settings page")]
+    [MetaCommand("saveToFile", "Save previous <count> submissions to <path>")]
```

Comment:
Saves n submissions in the REPL to a file, pretty straightforward and you should be able to understand this code